app/models/submission.rb

```diff
@@ -290,18 +290,19 @@ def self.get_satisfaction(satisfaction)
   end
 
   def self.get_location_data(params)
-    geocoder = Geocoder.search("#{params[:latitude]}, #{params[:longitude]}").first
+    Geocoder.configure(timeout: 15)
```

Comment:
I'm not really familiar with ruby, but I wanted to double check something here. Does ruby (or the Geocoder package) accept the syntax used here as well for hash parameters? If I'm looking at the right [Geocoder docs](https://www.rubydoc.info/github/alexreisner/geocoder/master/Geocoder.configure), it looks like this method expects a hash, something like this:

```ruby
Geocoder.configure(:timeout => 15)
```

Rather than this:

```ruby
Geocoder.configure(timeout: 15)
```

Reply:
Good question. I know it works because during testing I set it to 0.1 and confirmed I was getting timeout errors. 

See: https://stackoverflow.com/questions/16576477/how-do-methods-use-hash-arguments-in-ruby

Reply:
Cool, thanks for the link and clarification!